scripts/scenarios.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+
+# really would like to modify testrpc to add:
+#   - account params from a JSON file
+#   - `testrpc start` and `testrpc stop`, so these custom scripts don't
+#      have to be added to every project
+
+output=$(nc -z localhost 8545; echo $?)
+[ $output -eq "0" ] && ganache_running=true
+if [ ! $ganache_running ]; then
+  echo "Starting our own ganache instance"
+  ganache-cli \
+    -l 7000000 \
```

Comment:
is this gas limit?

Reply:
yep, I think this came from the setup we had before the proxy optimizations